### DISCUSSION TITLE
No DXVK Options

### DISCUSSION THREAD

**Discussion by DroofZA** [Whisky-App/Whisky#90] [General]
Did not get any DXVK options, what did I do wrong? (Sonoma, M2 Pro)

**Comment by IsaacMarovitz** [comment]
GPTK does not use DXVK, it was removed in 0.2.0

**Comment by DroofZA** [comment]
Oh, I was getting a "unable to initialise graphics, please ensure display drivers are up to date" when running any battle.net games. Thought that was related to the missing DXVK.

I also get no Wine pre-loader options like the screenshot. Any ideas?
